services/grpc/pom.xml

```diff
@@ -0,0 +1,63 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>geowave-service-parent</artifactId>
+		<groupId>mil.nga.giat</groupId>
+		<relativePath>../</relativePath>
+		<version>0.9.7-SNAPSHOT</version>
+	</parent>
+	<artifactId>geowave-service-grpc</artifactId>
+	<name>GeoWave gRPC services</name>
+	<dependencies>
+		<dependency>
+			<groupId>mil.nga.giat</groupId>
+			<artifactId>geowave-deploy</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>mil.nga.giat</groupId>
+			<artifactId>geowave-grpc-protobuf-shaded</artifactId>
+			<version>0.0.1-SNAPSHOT</version>
```

Comment:
shouldn't this be `${project.version}` now that the shaded artifact is part of the same versioning as the rest of geowave?